borgmatic/commands/borgmatic.py

```diff
@@ -123,23 +125,33 @@ def run_configuration(config_filename, args):  # pragma: no cover
     '''
     logger.info('{}: Parsing configuration file'.format(config_filename))
     config = validate.parse_configuration(config_filename, validate.schema_filename())
-    (location, storage, retention, consistency, hooks) = (
+    (location, storage, retention, consistency, hooks, metrics) = (
         config.get(section_name, {})
-        for section_name in ('location', 'storage', 'retention', 'consistency', 'hooks')
+        for section_name in ('location', 'storage', 'retention', 'consistency', 'hooks', 'metrics')
     )
 
     try:
         local_path = location.get('local_path', 'borg')
         remote_path = location.get('remote_path')
         borg_create.initialize_environment(storage)
 
+        if metrics and not args.dry_run:
+            stream = StringIO()
+            sh = logging.StreamHandler(stream)
+            logging.getLogger('borg_output').addHandler(sh)
+            logging.getLogger('borg_output').setLevel(logging.INFO)
```

Comment:
Because I always want to be able to log metrics (unless it's a `dry-run`), no matter if it's `-v1`, `-v2` or neither.

All the output from borg_output is just going directly into the metrics-module. Not to the default logger whichs namespace is `borgmatic`